client/src/pages/Login.js

```diff
@@ -56,10 +61,71 @@ const SocialButton = styled(Button).attrs((props) => ({
   margin: 0.5rem;
 `;
 
+const useStyles = makeStyles(theme => ({
+  container: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  textField: {
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(1),
+    width: '100%',
+  },
+  resize:{
+    fontSize: 20
+  },  
+}));
+
 export default ({ isLoginForm }) => {
-  const loginWithEmail = () => {
-    document.getElementById("login-password").submit();
-  };
+
+  const classes = useStyles();
+  const alert = useAlert();
+  const dispatch = useDispatch();
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleInputChangeEmail = e => {
+    setEmail(e.target.value);
+  }
+
+  const handleInputChangePassword = e => {
+    setPassword(e.target.value);
+  }
+
+  const handleLoginWithEmail = (evt) => {
+    evt.preventDefault();
+    if (!validateEmail (email)) {
+      alert.show('Invalid email address!', {
+        type: 'error',
+      });
+      return;
+    }
+    if (password.length < PASSWORD_MIN_LENGTH) {
+      alert.show('Password must be at least 6 characters', {
+        type: 'error',
+      });
+      return;
+    }
+    dispatch(loginWithEmail({ email, password }));
+  }
+
+  const handleSignup = (evt) => {
+    evt.preventDefault();
+    if (!validateEmail (email)) {
+      alert.show('Invalid email address!', {
+        type: 'error',
+      });
+      return;
+    }
+    if (password.length < PASSWORD_MIN_LENGTH) {
```

Comment:
we have to make sure we are using the same validation rules on backend as well

Reply:
yea sure. can you provide me the backend validation rules?

Reply:
seems like it's Auth0 validation rules

Reply:
Currently set to "Good" password strength settings : 

    No more than 2 identical characters in a row
    Special characters (!@#$%^&*)
    Lower case (a-z), upper case (A-Z) and numbers (0-9)
    Must have characters in length
    Non-empty password required

Reply:
@mannykary any idea if this is ok? We can also set it to "excellent" and then there is an extra rule:
No more than 2 identical characters in a row